src/backend/web/routes/posts.js

```diff
@@ -0,0 +1,53 @@
+require('../../lib/config');
+const express = require('express');
+const { getPost, getPosts } = require('../../utils/storage');
+
+const posts = express.Router();
+
+const env = process.env.NODE_ENV || 'development';
+const port = process.env.PORT || 3000;
+
+/**
+ * addGuidToPost inserts the redis key into the fetched post from redis.
+ * This is necessary since 'getPost' in 'storage.js' returns a post with no guid
+ */
+const addGuidToPost = async guid => {
+  const post = await getPost(guid);
+
+  /**
+   * If the post retrieved from redis doesn't have the property 'author'
+   * it means the id passed to addGuidToPost is invalid,
+   * A JSON { 'id': '-1' } is returned
+   */
+  post.guid = Reflect.has(post, 'author') ? guid : '-1';
+
+  return post;
+};
+
+posts.get('/', async (req, res) => {
+  const arrayOfPosts = [];
+
+  /**
+   * Temporary Date() until it is decided how to handle this
```

Comment:
Let's alter the `getPosts()` function to accept a different number of arguments, and make some optional.  We don't always need/want to limit the range by a date, but we might.  If we do, we should expose more route options here to pass in ranges.  For now, you could default to return all.